lib/simulator-xcode-6.js

```diff
@@ -174,11 +174,14 @@ class SimulatorXcode6 {
       return path.resolve(this.getDir(), s);
     });
 
-    let existence = await asyncmap(files, async (f) => { return fs.hasAccess(f); });
-
-    existence = await B.all(existence); // will throw an error if an fs.stat call fails
+    let existences = await asyncmap(files, async (f) => { return fs.hasAccess(f); });
+    for (let i = 0; i < files.length; i++) {
```

Comment:
rather than using an iterative forloop, this could be done using Array.forEach, which gives u value and array index